scripts/dist.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+# exist on error
+set -e
+VERSION=$(grep '"version"' package.json | sed 's/.*"version": *"\([^"]*\)".*/\1/')
+
+# echo commands
+set -x
+
+grunt clean
+grunt deploy
+# prevent ._ files from being included in the archive on macOS
+tar -czv --no-mac-metadata --no-acls -f gi-v${VERSION}.debug.tgz dist
```

Comment:
These options aren't portable.

For example:

```
tar: unrecognized option '--no-mac-metadata'
Try 'tar --help' or 'tar --usage' for more information.
```

Maybe using `find` along with `cpio` would have the desired effect.

For example,

```
find dist -not -name '._*' | cpio -H ustar -o - | gzip -9 > gi-v${VERSION}.debug.tgz
```